Source/JavaScriptCore/b3/air/AirArg.h

```diff
@@ -1396,7 +1412,7 @@ class Arg {
         case CallArg:
             return isValidAddrForm(opcode, offset(), width);
         case Index:
-            return isValidIndexForm(scale(), offset(), width);
```

Comment:
isValidIndexForm should also take op ode instead

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/da11283e6bd1aaa4c73f84c839f1a7b72f508ef3